packages/analytics-js/src/app/RudderAnalytics.ts

```diff
@@ -155,6 +231,8 @@ class RudderAnalytics implements IRudderAnalytics<IAnalytics> {
 
     // Process load method if present in the buffered requests
     if (loadEvent.length > 0) {
+      // Track page loaded lifecycle event if enabled
+      this.trackPageLifecycleEvents(preloadedEventsArray, loadEvent[3]);
```

Comment:
The load options normalization, etc., will happen in the load API of the Analytics class, whereas we're relying on that data (in the track life cycle function) before the load is even initiated.
Please double-check it.

Reply:
Addressed